algorithms/face_recognition/vectorize.py

```diff
@@ -1,23 +1,47 @@
-import json
-import face_recognition as fr
+from typing import Iterable
+from typing import List
+from typing import Optional
 
+from face_recognition import face_encodings
+from face_recognition import face_locations
+from face_recognition import load_image_file
+import numpy as np
+
+FaceVector = List[float]
+Image = np.array
+
+
+def get_face_embedding(face: Image) -> Optional[FaceVector]:
+    cropped_features = face_encodings(face)
+    if not cropped_features:
+        return None
+
+    face_vector = cropped_features[0]
+    return face_vector.tolist()
+
+
+def find_faces(img: Image) -> Iterable[Image]:
+    for top, right, bottom, left in face_locations(img):
+        yield img[top:bottom, left:right]
```

Comment:
Should we try replacing the face_recognition face finding with MTCNN for consistency across algorithms? We can use this pip installable package (https://github.com/ipazc/mtcnn).

Reply:
I'd rather keep this one simple for consistency with the ICRC v1 deployment so that we have a true baseline for comparison.